in_memory_document_session_operations.go

```diff
@@ -344,7 +344,7 @@ func (s *InMemoryDocumentSessionOperations) DeleteWithChangeVector(id string, ex
 	var changeVector *string
 	documentInfo := s.documentsById.getValue(id)
 	if documentInfo != nil {
-		newObj := s.entityToJson.convertEntityToJson(documentInfo.getEntity(), documentInfo)
+		newObj := EntityToJson_convertEntityToJson(documentInfo.getEntity(), documentInfo)
```

Comment:
What is the logic that guide you in choosing one or the other when building this api?

Reply:
Go doesn't have static methods on structs so in the port I settled for stand-alone function `Foo_bar()` so that we can distinguish `bar()` static function in 2 different classes.

This change is because I missed that `convertEntityToJson` is a static function before.

That being said it's not very Go-ish naming but at the moment I don't have a better scheme.

Reply:
ok